pkgs/development/libraries/ffmpeg/generic.nix

```
@@ -355,6 +355,13 @@ stdenv.mkDerivation (finalAttrs: {
         url = "https://git.ffmpeg.org/gitweb/ffmpeg.git/patch/814178f92647be2411516bbb82f48532373d2554";
         hash = "sha256-FQV9/PiarPXCm45ldtCsxGHjlrriL8DKpn1LaKJ8owI=";
       }
+    )
+    ++ (lib.optional (stdenv.isDarwin && lib.versionAtLeast version "6.1" && lib.versionOlder version "6.2")
```

```suggestion
    ++ (lib.optional (lib.versionAtLeast version "6.1" && lib.versionOlder version "6.2")
```
otherwise we likely notice it late when the patch doesn't apply anymore

---

I can make it unconditional, but it will cause a ton of rebuilds on Linux. Unless staging-next hasn’t made much progress, or causing unnecessary rebuilds isn’t an issue here, would it be better to do a follow up PR against staging with to make it apply unconditionally?

---

Oh, forgot that but doing a follow up would be great to avoid that linux people forgot to worry about the patch.